packages/collector/test/test_util/ProcessControls.js

```diff
@@ -61,7 +61,9 @@ class ProcessControls {
           const esmApp = files.find(f => f.indexOf('.mjs') !== -1);
 
           if (esmApp) {
-            opts.execArgv = [`--experimental-loader=${path.join(__dirname, '..', '..', 'esm-loader.mjs')}`];
+            opts.execArgv = isLatestEsmSupportedVersion(process.versions.node)
```

Comment:
issue: Our ESM pipeline for the collector runs with a single Node.js version. What is your plan to ensure both styles are covered in our pipelines?

Reply:
Discussed in offline, we will try to keep separate esm builds for 2 different Node.js versions for now, will check more available options.